gammapy/spectrum/tests/test_fit.py

```diff
@@ -110,10 +91,10 @@ def test_wstat(self):
                           stat='wstat', forward_folded=False)
         fit.fit()
         assert_allclose(fit.result[0].model.parameters['index'].value,
-                        1.997344538577775)
+                        1.9973633924241248)
```

Comment:
@registerrier What is a good relative tolerance, here the changes sometimes are O(1e-3) but for most code changes I would expect the changes well below this

Reply:
I would suggest this: usually put a reference number with high precision, 8 digits.
If it's a low-precision result, manually choose a lower rtol or atol. In this case rtol=1e-3 or rtol=1e-2 might be a good choice.

I explained this in the dev docs a few days ago here:
http://docs.gammapy.org/dev/development/howto.html#assert-convention

Reply:
I've also asked myself the question how to choose a good `rtol` for tests in the past. In some cases one could even make a "physical" choice. E.g. for spectral indices a relative error of `0.01` is perfectly fine, because systematics will always be larger...but I don't really have a suggestion.

Reply:
I won't add a tolerance for now, because I already changed the test values. But I will keep it in mind for the future